### PR TITLE
✨ new dto and service invocation

### PR DESCRIPTION
**Pull request by Shadowrunner11**

### CODE REVIEW

File: modules/com.pricning.cars/src/main/java/com/pricning/cars/portlet/ComPricningCarsPortlet.java [hunk 1]
```diff
@@ -47,17 +48,33 @@ public class ComPricningCarsPortlet extends MVCPortlet {
 
 	private static final RestTemplateImplemetation restTemplate = new RestTemplateImplemetation();
 
-	@Override
-	public void render(RenderRequest renderRequest, RenderResponse renderResponse) throws IOException, PortletException {
-		CarsResponseDTO carsInfo = restTemplate.serviceInvocation(
```
Comment (Brandt322) [inline modules/com.pricning.cars/src/main/java/com/pricning/cars/portlet/ComPricningCarsPortlet.java:52]: Delete line 52

File: gradle.properties [hunk 2]
```diff
@@ -8,4 +8,3 @@ liferay.workspace.themes.dir=themes
 liferay.workspace.wars.dir=modules
 microsoft.translator.subscription.key=
 liferay.workspace.product=dxp-7.2-sp2
-liferay.workspace.home.dir=/home/piero/Downloads/liferay-dxp-7.2.10.2-sp2
```
Comment (Brandt322) [inline gradle.properties]: Move to your local gradle

File: modules/com.pricning.cars/src/main/java/com/pricning/cars/portlet/ComPricningCarsPortlet.java (hunk 1 above)
Comment (Shadowrunner11) [reply modules/com.pricning.cars/src/main/java/com/pricning/cars/portlet/ComPricningCarsPortlet.java:52]: solved

File: gradle.properties (hunk 2 above)
Comment (Shadowrunner11) [reply gradle.properties]: solved

**Review by Brandt322 — APPROVED** [review]

**Pull request merged**